server/src/units/registerUser.ts

```diff
@@ -3,10 +3,15 @@ import { gql } from 'apollo-server-express'
 import { logger } from '@shared'
 
 const mutation = gql`
-  mutation registerUser($authServerId: String!, $emailPrimary: String!) {
+  mutation registerUser($authServerId: String!, $emailPrimary: String!, $firstName: String, $lastName: String, $middleName: String,$displayFullName: String, $citationName: String) {
     insert_users(objects: {
       auth_server_id: $authServerId,
       email_primary: $emailPrimary,
+      first_name: $firstName,
+      last_name: $lastName,
+      middle_name: $middleName,
+      display_full_name: $displayFullName,
+      citation_name: $citationName
```

Comment:
Remember, we're moving to camelCase for field names in the database. We're also using schema.org names when possible (e.g., givenName, familyName, additionalNames). Citation name shouldn't be necessary as, I thought, we were deriving this from the given, family, and additional.

Reply:
Changed the columns name to camelCase according to schema.org